src/audio/CMakeLists.txt

```diff
@@ -208,7 +208,7 @@ set(mixer_sources ${mixer_src})
 set(src_sources src/src.c src/src_generic.c)
 set(asrc_sources asrc/asrc.c asrc/asrc_farrow.c asrc/asrc_farrow_generic.c)
 set(eq-fir_sources eq_fir/eq_fir.c eq_fir/eq_fir_generic.c)
-set(eq-iir_sources eq_iir/eq_iir.c)
+set(eq-iir_sources module_adapter/module_adapter.c module_adapter/module/generic.c eq_iir/eq_iir.c)
```

Comment:
don't we now add these in src/audio/module_adapter/CMakeLists.txt?

Reply:
... there's indeed some CONFIG_COMP_FIR logic in module_adapter CMakeLists.txt.